scalableminds/webknossos
# Create Annotation From View Mode: Keep Activated Mapping

### URL of deployed dev instance (used for testing):
- https://createexplorationalmapping.webknossos.xyz

### Steps to test:
- Create annotation from dashboard, should work as before
- Create annotation from view mode (color layer only), should work as before
- Create annotation from view mode (with segmentation with no mapping active), should work as before
- Create annotation from view mode (with segmentation and hdf5 mapping selected), annotation should have that mapping active in the fallback segmentation

### Issues:
- fixes #6645

------
